mission_control/src/mission_control.cpp

```diff
@@ -203,66 +133,59 @@ void MissionControlNode::reportHeartbeat(const ros::TimerEvent& timer)
   pub_report_heartbeat.publish(outmsg);
 }
 
+void MissionControlNode::executeMissionT(const ros::TimerEvent& timer)
+{
+  NodeStatus missionStatus = m_mission_map[m_current_mission_id]->getMissionStatus();
+  if (missionStatus == NodeStatus::IDLE || missionStatus == NodeStatus::RUNNING)
+  {
+    missionStatus = m_mission_map[m_current_mission_id]->executeMissionTickEvent();
+  }
+  else
+  {
+    m_mission_map[m_current_mission_id]->stopMission();
```

Comment:
@patapufate does the mission need to be stopped if it's not running?

Reply:
Nop. You're right.